cmd/aries-js-worker/main.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"syscall/js"
+	"time"
+)
+
+// command is received from JS
+type command struct {
+	ID      string `json:"id"`
+	Pkg     string `json:"pkg"`
+	Fn      string `json:"fn"`
+	Payload string `json:"payload"`
+}
+
+// result is sent back to JS
+type result struct {
+	ID      string `json:"id"`
+	IsErr   bool   `json:"isErr"`
+	ErrMsg  string `json:"errMsg"`
+	Payload string `json:"payload"`
+}
+
+// All supported command handlers.
+func handlers() map[string]map[string]func(*command) *result {
+	return map[string]map[string]func(*command) *result{
+		"test": {
```

Comment:
I assume we will load handlers from the responsible package in a follow-up PR?

Reply:
Will think about it more in the follow-up PR.